web-client/src/presenter/actions/getCaseAssociationAction.test.js

```diff
@@ -155,7 +155,9 @@ describe('getCaseAssociation', () => {
       props: {},
       state: {
         caseDetail: {
-          userId: '123',
+          contactPrimary: {
```

Comment:
Should the test case below this one be updated similarly, to have a contactId in the caseDetail instead of a userId?

Reply:
Modifications made in relevant test at 186.